pkg/backup/pod_action.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 the Velero contributors.
+Copyright 2017, 2022 the Velero contributors.
```

Comment:
```suggestion
Copyright the Velero contributors.
```

https://velero.io/docs/v1.8/code-standards/#copyright-header

Same as belows.

Reply:
Look like the new copyright format by github and Velero community.  I'll fix it.